ee_download/zonal.py

```diff
@@ -109,12 +120,13 @@ def run_data_2018_baseline():
 		liq = dataset["liq"]
 		raster_folder = dataset["raster_folder"]
 		output_folder = dataset["output_folder"]
-		rasters = [item for item in os.listdir(raster_folder) if item.endswith(".tif") and item not in skips]  # was going to do this differently, but leaving it aloine
+		rasters = [item for item in os.listdir(raster_folder) if item.endswith(
```

Comment:
same linesplit problem. Feel free to split, but do it with a slash at a better spot